functions/cache.test.ts

```diff
@@ -0,0 +1,30 @@
+import CacheMock from 'browser-cache-mock'
```

Comment:
Can this be moved to be colocated with the file it tests: functions/utils/cache.test.ts?